src/index.ts

```diff
@@ -10,28 +11,59 @@ const typeDefs = `#graphql
   type Query {
     hello: [SimpleText]
   }
+
+  type Mutation  {
+    createUser(data: UserInput!): User
+  }
+
+  type User {
+    id: ID!
+    name: String
+    email: String
+    password: String
+    birthDate: String 
```

Comment:
```suggestion
    id: ID!
    name: String!
    email: String!
    birthDate: String
```

Sobre a sugestão:

1. Id, nome e e-mail poderia ser considerados obrigatórios, enquanto birthDate poderia ser considerado opcional (só uma forma de você trabalhar com os 2 tipos de campo no onboard)
2. Retornar um campo chamado "password" na mutation não é uma boa, independente do valor.

Reply:
Sobre essas alterações, elas também devem ser feitas no `schema.prisma`?

Reply:
O banco retornar senha não tem problema, porque estando barrado no GraphQL, o dado não retornará na response da chamada. Sobre a obrigatoriedade dos campos, é bom sim seguir o mesmo esquema da API.